.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: 🐛 Bug
+
+description: Report an issue to help improve the project.
+
+labels: ["🛠 goal: fix"]
```

Comment:
Does this mean the issue labels?
if so, it should be labelled as `bug`, instead of create a new label

Reply:
no it means that this report helps in bug fix

Reply:
it automatically attach goal: fix label but i think you are right i corrected that now it will label it bug